doc/user_doc/vic_admin/delete_vch_options.md

```diff
@@ -43,6 +43,21 @@ Wrap the password in single quotation marks (') on Mac OS and Linux and in doubl
 
 <pre>--password '<i>esxi_host_or_vcenter_server_p@ssword</i>'</pre>
 
+### `thumbprint` ###
```

Comment:
It'll also be too painful to try to do this for 0.7. Opened https://github.com/vmware/vic/issues/2978 to examine whether we can do this for GA. If we were in DITA, it would be easy to single-source the doc and reuse it in each command, but since we're in Markdown it's all manual and so updates are indeed a maintenance headache.